frontend/src/container/InfraMonitoringK8s/Clusters/utils.tsx

```diff
@@ -136,14 +136,27 @@ export const getK8sClustersListColumns = (
 	return columnsConfig as ColumnType<K8sClustersRowData>[];
 };
 
+const dotToUnder: Record<string, keyof K8sClustersData['meta']> = {
+	'k8s.cluster.name': 'k8s_cluster_name',
+	'k8s.cluster.uid': 'k8s_cluster_uid',
+};
+
 const getGroupByEle = (
 	cluster: K8sClustersData,
 	groupBy: IBuilderQuery['groupBy'],
 ): React.ReactNode => {
 	const groupByValues: string[] = [];
 
 	groupBy.forEach((group) => {
-		groupByValues.push(cluster.meta[group.key as keyof typeof cluster.meta]);
+		const rawKey = group.key as string;
+
+		// 3) Choose mapped key if present, otherwise use rawKey
```

Comment:
Remove the numbering.
Overall please evaluate all the added comments and see which are actually required or need to be modified